config/locales/server.en.yml

```diff
@@ -7,6 +7,10 @@ en:
         invalid_field: Field’s component `%{component}` is not usable on `%{target}:%{target_name}`
         invalid_metadata: Data for `%{field}` is invalid or component `%{component}` is unknown.
     triggerables:
+      api_call:
+        title: API call
+        description: Making a POST request to trigger an automation
+        doc: The POST request should be made to `/automations/:id/trigger.json` with a global API key or a User API key with the `discourse_automation:automations_trigger` scope.
```

Comment:
The scope `discourse_automation:automations_trigger` is not correct. In core, when the `add_user_api_key_scope` code is run the `prefixed_scope_name` ends up being `discourse-automation:automations_trigger`:

https://github.com/discourse/discourse/blob/bbf47345f8b44de0e8fda5627fb9c9cc5c917ab9/lib/plugin/instance.rb#L852-L861

This is because the `name` from the plugin `metadata` is `discourse-automation`

Reply:
Yes you are write that's a typo, I got it right in the spec